### PR TITLE
Bug 1958080: Add missing error page strings

### PR DESCRIPTION
**Pull request by jhadvig**
Sent the translations to the localization team, with help of @rebeccaalpert ...will update once they come.

/assign @stlaz 

cc'ing @sg00dwin

### CODE REVIEW

File: pkg/server/login/login.go [hunk 1]
```diff
@@ -134,7 +134,7 @@ func (l *Login) handleLoginForm(w http.ResponseWriter, req *http.Request) {
 	form.Locale = locales.GetLocale(req.Header.Get("Accept-Language"))
 	form.ErrorCode = req.URL.Query().Get(reasonParam)
 	if len(form.ErrorCode) > 0 {
-		if msg, hasMsg := errorMessages[form.ErrorCode]; hasMsg {
+		if msg, hasMsg := form.Locale[errorMessages[form.ErrorCode]]; hasMsg {
```
Comment (stlaz) [inline pkg/server/login/login.go:137]: what happened here?

**Comment by jhadvig** [comment]
Successfully verified that the fix is working. Removing `hold`. 

Screenshot 2021-05-18 at 17 32 03


/hold cancel

**Pull request merged**